ts/twitter/src/lib/sdk/rabbitmq-logger-sdk/rabbitmq-logger.ts

```diff
@@ -0,0 +1,42 @@
+import amqp from 'amqplib';
+
+export class RabbitMQLogger {
```

Comment:
Sorry, I forgot to mention that.
Please correct the other parts as well.
```suggestion
export class RabbitMqLogger {
```

FYI: https://google.github.io/styleguide/tsguide.html#camel-case